src/Administration/Resources/app/administration/src/app/component/media/sw-media-upload-v2/index.js

```diff
@@ -187,7 +186,7 @@ export default {
             preview: null,
             isDragActive: false,
             defaultFolderId: null,
-            isUploadUrlFeatureEnabled: false,
+            isUploadUrlFeatureEnabled: Shopware.Store.get('context').app.config?.settings?.enableUrlFeature ?? true,
```

Comment:
Fallback should be `false` instead of `true` to behave like before

Reply:
u sure jest tests says something different